Orange/tests/test_domain.py

```diff
@@ -340,7 +295,7 @@ def test_has_discrete(self):
         self.assertFalse(Domain([]).has_discrete_attributes())
         self.assertFalse(Domain([], [age]).has_discrete_attributes())
         self.assertFalse(Domain([], race).has_discrete_attributes())
-
+        
```

Comment:
This looks like a blank space. You can set PyCharm to strip trailing whitespace on modified lines in Settings > Editor > General.